src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -179,11 +179,11 @@ def arm_exception_handler(ex, fault_type, summary, return_if_not_found=False):
         if status_code // 100 == 4:
```

Comment:
Add one more condition for checking 5xx response codes and classify that as AzureInternalError